src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -199,24 +198,6 @@ public synchronized static NewGameChooserModel getNewGameChooserModel() {
       return new NewGameChooserModel();
   }
 
-
-  /**
-   * Refreshes the game list (from disk) then caches the new list
-   */
-  private void refreshGameList() {
```

Comment:
No. Everything is called elsewhere except for the enabled, which is not necessary since the jlist is enabled by default.